examples/index.md

```diff
@@ -178,64 +180,35 @@ Content-Type: application/vnd.api+json
 {
   "errors": [
     {
+      "status": "422",
+      "source": { "pointer": "/data/attributes/first-name" },
       "title":  "Invalid Attribute",
-      "source": { "pointer": "data/attributes/first-name" },
       "detail": "First name must contain at least three characters."
     }
   ]
 }
 ```
 
-Notice that the attribute `first-name` is dasherized.  Dasherized attributes are preferred
```

Comment:
I removed this note about dasherization because it wasn't specific to errors and I wanted to keep this section (which is already, by necessity, pretty long, as short as possible.)